sa/db-next/boulder_sa/20250417000000_RateLimitOverrides.sql

```diff
@@ -0,0 +1,20 @@
+-- +migrate Up
+-- SQL in section 'Up' is executed when this migration is applied
+
+CREATE TABLE overrides (
+   `limitEnum` tinyint(4) UNSIGNED NOT NULL,
+   `bucketKey` varchar(255) NOT NULL,
+   `comment`   varchar(255) DEFAULT NULL,
```

Comment:
IMO this could easily be NOT NULL; it makes handling the model type a bit easier and we likely want a comment for every single entry.